docs/requirements.md

```diff
@@ -1,28 +1,37 @@
-On the other hand, when you perform all operations against a single master instance, the consistency will be linearizable.
-We recommend performing all operations/transactions against a single master instance (so that you can achieve linearizable) if you want to avoid caring consistency issues in your applications. Note that you can still use a read replica as a backup and standby even if you follow the recommendation.
+ScalarDB does not work on some Cassandra-compatible databases that do not support LWTs, such as [Amazon Keyspaces](https://aws.amazon.com/keyspaces/). This is because the Consensus Commit transaction manager relies on the linearizable operations of underlying databases to make transactions serializable.
 
-If you strongly want to perform operations/transactions on read replicas, you can configure your application to perform read-write-mixed transactions against a master instance and read-only transactions against read replicas.
-The resulting schedule would not be strict serializable (linearizable and serializable) but would be serializable.
+## CosmosDB database requirements
+
+In your Azure CosmosDB account, you must set the **default consistency level** to **Strong**.
+
+Consensus Commit, the ScalarDB transaction protocol, requires linearizable reads. By setting the **default consistency level** to **Strong**, CosmosDB can guarantee linearizability.
+
+For instructions on how to configure this setting, see the official documentation at [Configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#configure-the-default-consistency-level).
+
+## JDBC database recommendations
+
+In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code by using the `Operation.withConsistency()` method. In addition, the consistency level depends on the setup of your JDBC database.
+
+For example, if you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas. On the other hand, if you perform all operations against a single master instance, the consistency will be linearizable.
+
+We recommend performing all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. Note that you can still use a read replica as a backup and standby even if you follow this recommendation.
+
+If you must perform operations or transactions on read replicas, you can configure your application to perform read-write-mixed transactions against a single master instance and read-only transactions against read replicas. The resulting transaction schedules with the configuration would not be strict serializable (linearizable and serializable) but would be serializable.
```

Comment:
```suggestion
Also, from the perspective of the specification of ScalarDB, the write operation sometimes occurs against the underlying database to update metadata for transaction management even if the transaction is read-only. Therefore, you cannot use read replica under ScalarDB. Note that you can still use a read replica as a backup and standby even if you follow this recommendation.
```
Before we discussed `whether ScalarDB supports read replica or not`. And, if my remembering is correct, as a result of the discussion, we decided that we don't support read replicas.

Note: You can see the result in the following issue in Jira.
https://scalar-labs.atlassian.net/browse/DLT-13744

So, I left a suggestion to apply the above things to this document.

@josh-wong 
Could you please take a look at my suggestion from a grammatical point of view?

@feeblefakie @brfrn169 
Could you please take a look at my suggestion from a technical point of view?

Reply:
@kota2and3kan You are right. We can't use read replicas when using `EXTRA_WRITE`.

Reply:
@brfrn169 
> You are right. We can't use read replicas when using EXTRA_WRITE. Thanks.

Thank you for your confirmation! I will update this sentence to make it simpler (We only state `read replica is not supported). 

So, in this document, I don't mention concrete things like `EXTRA_WRITE` since it might lead to confusion.

@feeblefakie  
> For how to correct it,
> how about simply saying something like this?
> We can put the reason as a side note or something, but I think we should not start with reasoning.

Ah, I see. Indeed, describing too concrete internal information may cause confusion for users.

I agree with your suggestion that describe it simply. I will update the sentence that I suggested.

Reply:
@feeblefakie @brfrn169 @josh-wong 
```suggestion
With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. Note that you can still use a read replica as a backup and standby even if you follow this guideline.
```

I updated my suggestion!

* Remove concrete (internal) reasons.
* Add `ScalarDB does not support read replicas` explicitly to make requirements clear and avoid unnecessary inquiries from users.
* Explain `You can still use a read replica as a backup and standby` as a note.

Please take a look again when you have time!

Reply:
@kota2and3kan Thank you! It looks good.
@josh-wong Please review this from an English perspective.

Reply:
LGTM! The only change I made was to put the "Note that ..." part into a notice block.

```suggestion
With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 

{% capture notice--info %}
**Note**

You can still use a read replica as a backup and standby even when following this guideline.
{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Reply:
> The only change I made was to put the "Note that ..." part into a notice block, which I should have done earlier.

Looks good! Thank you!